src/fragments/lib/predictions/js/sample.mdx

```diff
@@ -5,11 +5,14 @@ A sample React application with all of the Predictions features is provided belo
 To use `Predictions.convert()` with Amazon Transcribe you will need to install the following dependency in your app first:
 
 ```bash
-npm install microphone-stream
+yarn add microphone-stream
 ```
 
+If you are running into an issue with `Uncaught ReferenceError: Buffer is not defined` due to `react-scripts@5.0.0`, try following the instructions provided in [this stack exchange answer](https://stackoverflow.com/a/71205013).
+
```

Comment:
Is there a better way to address this? We currently suggest that customers use an older library, which only works if you do some shenanigans with your react setup as described in the stack overflow comment linked above.